lib/perl/Genome/VariantReporting/Command/Wrappers/ModelPair.t

```diff
@@ -14,11 +14,22 @@ use File::Slurp qw(read_file);
 use Genome::VariantReporting::Command::Wrappers::TestHelpers qw(get_build compare_directories);
 use Genome::VariantReporting::Framework::TestHelpers qw(test_dag_xml);
 use Genome::Utility::Test qw(compare_ok);
+use Sub::Install qw(reinstall_sub);
 
 my $pkg = "Genome::VariantReporting::Command::Wrappers::ModelPair";
 
 use_ok($pkg);
 
+reinstall_sub({
```

Comment:
It doesn't really matter since the scope ends up being the same in this case but `Sub::Override` is probably a slightly better fit for overriding subroutines in tests.